src/dragonball/src/metric.rs

```diff
@@ -60,6 +61,8 @@ pub struct DragonballMetrics {
     pub seccomp: SeccompMetrics,
     /// Metrics related to signals.
     pub signals: SignalMetrics,
+    /// Metrics related to balloon device.
+    pub balloon: HashMap<String, Arc<BalloonDeviceMetrics>>,
```

Comment:
Could you add the comment here that `String` refer to balloon id.

Reply:
I added a comment to explain that the key of HashMap is `DeviceId`.